raiden-ts/src/path/epics.ts

```diff
@@ -89,7 +89,11 @@ export const pathFindServiceEpic = (
                   }),
                   map(
                     (results: PathResults): Paths =>
-                      results.result.map(r => ({ path: r.path, fee: r.estimated_fee })),
+                      results.result.map(r => ({
+                        path: r.path,
+                        // Add PFS safety margin to estimated fees
+                        fee: r.estimated_fee.mul(pfsSafetyMargin * 10).div(10) as Int<32>,
```

Comment:
This will break if receiving a `pfsSafetyMargin` with steps smaller than `0.1`, e.g. `1.125`. Use `Math.round`, floor or other integer-assuring function to ensure an integer is passed to `mul`, and use a bigger multiplier to ensure more granularity (e.g. 1e6 will allow granularity down to 1e-6, it can be too much, but it doesn't add complexity and at least 100 for percentages would be desired IMO):
```suggestion
                        fee: r.estimated_fee.mul(Math.round(pfsSafetyMargin * 1e6)).div(1e6) as Int<32>,
```

Reply:
Thanks, wasn't sure what kind of values for safety margins are going to be used here. Will fix